### ISSUE TITLE
Add descriptives of SES and % NAE overall and by lab to Table 1

### ISSUE THREAD

**Issue by kristabh** [manybabies/mb1b-analysis-public#3]
"Thank you for the thorough discussion of how bilingualism was assessed across the languages. However, there was much less detail about how the categorization of NAE was defined. In fact, I don’t think it was ever actually defined.  Some descriptives of SES and % NAE overall and by lab would be helpful. I have no sense of the background of the participants."

**Comment by jkosie** [comment]
I'm starting to work on this!

**Comment by jkosie** [comment]
I've edited table 1 to included average years of maternal education by lab and % NAE exposure by lab. I included only % NAE exposure for bilinguals since thats what is used in the analyses, but I can change it. I also sorted by % NAE (see issue #4).

The only problem is that now the table is large, so I had to turn it to landscape format. I'm not sure we want to do that. Feel free to suggest changes - I'm happy to make them!
